OpenRA.Game/Traits/TraitsInterfaces.cs

```diff
@@ -282,6 +282,12 @@ public interface IMove
 		bool CanEnterTargetNow(Actor self, Target target);
 	}
 
+	public interface ITemporaryBlocker
+	{
+		bool CanRemoveBlockage(Actor self, Actor blocking);
```

Comment:
This interface suggest the implementing trait (actor) is a blocker.
If so, why would they want to remove other blockage?